File: TransCoder_tokenized_test_set_functions/java/INTERCHANGE_ELEMENTS_OF_FIRST_AND_LAST_ROWS_IN_MATRIX.java
static void interchangeFirstLast ( int m [ ] [ ] ) {
  int rows = m . length;
  for ( int i = 0;
  i < m [ 0 ] . length;
  i ++ ) {
    int t = m [ 0 ] [ i ];
    m [ 0 ] [ i ] = m [ rows - 1 ] [ i ];
    m [ rows - 1 ] [ i ] = t;
  }
}

File: TransCoder_tokenized_test_set_functions/python/INTERCHANGE_ELEMENTS_OF_FIRST_AND_LAST_ROWS_IN_MATRIX.py
def interchangeFirstLast ( mat, n, m ) :
    rows = n
    for i in range ( n ) :
        t = mat [ 0 ] [ i ]
        mat [ 0 ] [ i ] = mat [ rows - 1 ] [ i ]
        mat [ rows - 1 ] [ i ] = t
    
